src/schemas/complaint-tool/schema.js

```diff
@@ -83,38 +82,35 @@ let schema = {
     'issueResolution'
   ],
   properties: {
-    address: {
-      type: 'object',
-      required: ['street', 'city', 'state', 'postalCode', 'country'],
-      properties: {
-        street: {
-          type: 'string',
-          minLength: 1,
-          maxLength: 1000 // TODO: confirm length limit with stakeholders 
-        },
-        street2: {
-          type: 'string',
-          minLength: 1,
-          maxLength: 1000 // TODO: confirm length limit with stakeholders 
-        },
-        city: {
-          type: 'string',
-          minLength: 1,
-          maxLength: 25
-        },
-        state: {  // TODO: verify validation as abbreviations or words
-          type: 'string',
-          'enum': GIBSFTStates
-        },
-        postalCode: {  // TYPE: text (5)
-          type: 'string',
-          pattern: '^\\d{5}$' // common definition pattern (meets submission requirements)
-        },
-        country: { // TODO: determine validation, picklist? 
-          type: 'string',
-          'enum': ['US'] // Only US addresses are supported
-        }
-      }
+    // TRANSLATE: address and address2 must be combined into a single comma separated string property named 'address'
```

Comment:
would it be useful to add an example of what the final combined `address` should look like given sample values for the input `address` and `address2` fields? Such as "`given 'address': '123 Main St.' and 'address2': 'APT 23' then the final translated 'address' should be '123 Main St., APT 23'`

Reply:
@lihanli are we planning to perform these translations on the BE? If so, would anything in addition to these "TRANSLATE" comments be helpful?

Reply:
an example should be fine

Reply:
Great idea @erikphansen, updated.